db/package.json

```diff
@@ -15,6 +15,7 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
+    "@binaris/shift-interfaces": "0.0.1",
```

Comment:
This intentionally the exact version. When interfaces version changes during publication also the db version changes, this allows us to know exactly what are the versions of dependant packages from the version of the parent instead of a combinatorial guess.

Note that even if interfaces change is major, the automatic change in db package will be patch.

Reply:
SGTM